src/logging.cc

```diff
@@ -161,6 +161,8 @@ static const char* DefaultLogDir() {
   return "";
 }
 
+GLOG_DEFINE_int32(logfile_mode, 0644, "Log file mode/permissions");
```

Comment:
Good catch on the default, o666 sounds good.

"some" flags have a period. If you prefer a period just ask :)